services/movie.py

```diff
@@ -1,11 +1,13 @@
+from django.db import transaction
 from django.db.models import QuerySet
 
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+        genres_ids: list[int] = None,
```

Comment:
Wrong type annotations must be
Optional[list[int]] because it accepts None as default
The same in all places

Reply:
With all types

Reply:
Thank you. Fixed in all places